buildman/manager/executor.py

```diff
@@ -552,7 +552,7 @@ def _build_job_containers(self, token, build_uuid):
 
         container = {
             "name": "builder",
-            "imagePullPolicy": "IfNotPresent",
+            "imagePullPolicy": "Always",
```

Comment:
As @kleesc  suggested, can we make this a config value?

Reply:
yes, updated to be configurable.